app/frontend/src/features/profile/view/ProfilePage.tsx

```diff
@@ -65,15 +73,21 @@ export default function ProfilePage() {
       ) : user ? (
         <ProfileUserProvider user={user}>
           <div className={classes.root}>
-            <Overview user={user} />
+            <Overview user={user} setIsRequesting={setIsRequesting} />
             <Card className={classes.detailsCard}>
               <TabContext value={currentTab}>
                 <TabBar
                   value={currentTab}
                   setValue={setCurrentTab}
                   labels={SECTION_LABELS}
-                  ariaLabel="tabs for user's details"
+                  ariaLabel={SECTION_LABELS_A11Y_TEXT}
                 />
+                <Collapse in={isRequesting}>
```

Comment:
This looks so great, and it's so easy! We should keep these transitions in mind.

Reply:
Definitely :D